### PR TITLE
docs update to promote TaintBasedEvictions to beta

### PR DESCRIPTION
**Pull request by Huang-Wei**
/sig scheduling
/milestone v1.13

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/federation-apiserver.md [hunk 1]
```diff
@@ -109,8 +109,8 @@ ServiceProxyAllowExternalIPs=true|false (DEPRECATED - default=false)
 StreamingProxyRedirects=true|false (BETA - default=true)
 SupportIPVSProxyMode=true|false (BETA - default=false)
 SupportPodPidsLimit=true|false (ALPHA - default=false)
-TaintBasedEvictions=true|false (ALPHA - default=false)
-TaintNodesByCondition=true|false (ALPHA - default=false)
+TaintBasedEvictions=true|false (BETA - default=true)
```
Comment (k82cn) [inline content/en/docs/reference/command-line-tools-reference/federation-apiserver.md]: federation api server?

Comment (Huang-Wei) [reply content/en/docs/reference/command-line-tools-reference/federation-apiserver.md]: I just searched all occurrences of `TaintBasedEvictions`. IMO enabling a featuregate in `kube_features.go` is populated to all components, right?

**Comment by AishSundar** [comment]
Thanks @Huang-Wei for the PR. @tfogo our 1.13 Docs lead as FYI

File: content/en/docs/reference/command-line-tools-reference/federation-apiserver.md (hunk 1 above)
Comment (k82cn) [reply content/en/docs/reference/command-line-tools-reference/federation-apiserver.md]: should be ; just not sure how federation handle such case: some feature upgrade from alpha to beta, does federation test them? maybe you can ping multicluster wg/sig .

**Pull request merged**